src/Microsoft.ML.CpuMath/AssemblyInfo.cs

```diff
@@ -2,8 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Reflection;
 using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
+using Microsoft.ML;
```

Comment:
>Microsoft [](start = 6, length = 9)

In order for the "best friends" logic to be in effect, you will need to have the `WantsToBeBestFriends` assembly level attribute here in this assembly (CpuMath), so that the checks actually happen. The way it is set up right now, the checks will not happen, you just made everything friendly (whether the things they're referencing are best friends or not).

Reply:
Fixed. Thanks.

---
In reply to: [234739025](https://github.com/dotnet/machinelearning/pull/1659#discussion_r234739025) [](ancestors = 234739025)